python/sunbeam/schedule.py

```diff
@@ -16,6 +16,22 @@ def wells(self):
     def group(self, name):
         return Group(self._group(name), self)
 
+    def group_tree(self, timestep):
+        return GroupTree(self._group_tree(timestep))
+
+    def gtree(self, timestep):
+        '''Fuction to return python struct of group topologi for given timestep'''
+        gt_obj = self._group_tree(timestep)
+        gt={}
+        for n in range (len(self.groups)):
+            group=self.groups[n].name
```

Comment:
`for group in self.groups:` instead of using `range`.  Also, please don't have a space between a function call and the parentheses, i.e. `range(x)` instead of `range (x)`.